.travis.yml

```diff
@@ -33,7 +31,6 @@ matrix:
       - rustup component add clippy
       - rustup target add $TARGET_32
      script:
-      - RUSTFLAGS="-D warnings" cargo build --verbose
```

Comment:
Is it related to cache?

Reply:
Ooops, no, this was an accident :/

Though now I wonder how useful this is, since `cargo package` basically already covers this. Maybe we should just remove this from the nightly build as well...

Reply:
Made https://github.com/cloudflare/quiche/pull/323 to do that. We can go the other way and restore this if you disagree though.

Reply:
Yes I think we can do this way too. Thanks.